jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerContextGenerator.java

```diff
@@ -46,12 +46,45 @@
  */
 public class DockerContextGenerator {
 
+  /**
+   * Formats a list for the Dockerfile's ENTRYPOINT or CMD.
+   *
+   * @see <a
+   *     href="https://docs.docker.com/engine/reference/builder/#exec-form-entrypoint-example">https://docs.docker.com/engine/reference/builder/#exec-form-entrypoint-example</a>
+   * @param items the list of items to join into an array.
+   * @return a string in the format: ["item1","item2",...]
+   */
+  @VisibleForTesting
+  static String joinAsJsonArray(List<String> items) {
+    return "["
```

Comment:
Oh for this we could actually use `ObjectMapper` from the JSON library we use (I forgot about it when implementing this). It will automatically generate the JSON string from a list of strings.

You could do like

`Blobs.writeToString(JsonTemplateMapper.toBlob(thelist))`

Reply:
I think it needs a `ListOfJsonTemplate`, not a list of strings?

Reply:
Oh right, maybe just use `new ObjectMapper().writeValueAsString` directly.